src/shared/modules/auth/common.js

```diff
@@ -161,9 +164,17 @@ export const getCredentialsFromAuthResult = (result, idpId) => {
     parsedJWT[principal] || parsedJWT.email || parsedJWT.sub
   authLog(`Credentials assembly with username: ${credsPrincipal}`)
 
+  const configuredTokenType =
+    selectedSSOProvider.config?.['token_type_authentication']
   const tokenTypeAuthentication =
-    selectedSSOProvider.config?.['token_type_authentication'] ||
-    defaultTokenTypeAuthentication
+    configuredTokenType || defaultTokenTypeAuthentication
+
+  if (!configuredTokenType) {
+    authLog(
```

Comment:
Since an earlier change was to make this `authLog` -> `authlog`, should that change not apply everywhere?

Reply:
That was made by accident and corrected to be authLog everywhere now